import_presubmit_config.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script for creating a CI presubmit configuration that can be used in the federation test pipeline.
+# It can either generate a standard configuration, or take the yaml configuration of a "regular" presubmit pipeline
+# and generate a modified version that can be used to test a project as part of the federation.
+# In both cases the script places the generated file in the right directory and registers the config in the master
+# config file.
+
+import argparse
+import codecs
+import os
+import re
+import sys
+import urllib.request
+import utils
+import traceback
+import yaml
+
+# TODO(fweikert): keep this list in sync with PLATFORMS in https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazelci.py
+# (or find a better solution)
+PYTHON_VERSIONS = {"debian10": "python3.7", "macos": "python3.7", "windows": "python.exe"}
```

Comment:
nit: line wrap and indent 4.

Reply:
The formatter doesn't like that change :(